vNext/src/ClientGenerator/BootstrapCodegen.proj

```diff
@@ -0,0 +1,48 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
well, previously they were distributed in Orleans.csproj, and also each project that was part of the bootstrap had a big section in it for when bootstrap was true that was copy/pasted in each. This is more centralized